invalidation-mode/src/main/java/org/infinispan/tutorial/simple/invalidation/InvalidationMode.java

```diff
@@ -0,0 +1,113 @@
+package org.infinispan.tutorial.simple.invalidation;
+
+import java.util.Scanner;
+
+import org.infinispan.Cache;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfiguration;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.manager.DefaultCacheManager;
+
+public class InvalidationMode {
+
+   // Run this main at least twice and test how get/put/putForExternalRead/remove behave across nodes
```

Comment:
This sounds like a comment in a test... for a tutorial I'd expect a javadoc on the class enumerating the expectations:

* put should insert the value in one node and remove it in the others
* pfer should insert the value in one node and leave it in the others
* remove should remove it everywhere
* get should read the value from the local node

Maybe also add an option to configure a `ClusterLoader`, in which case get should read the value from any node, not just the local one.

Reply:
Ok for the JavaDoc. For the ClusterLoader I will see, but I don't want to overcomplicate more since this is only a simple tutorial